compiler/qsc/src/interpret.rs

```diff
@@ -315,6 +318,70 @@ impl Interpreter {
         })
     }
 
+    /// Given a package ID, returns all the global items in the package.
+    /// Note this does not currently include re-exports.
+    pub fn get_global_items(
```

Comment:
Per rust naming conventions I'd suggest removing the `get_` prefix from these methods

Reply:
Huh... as it happens, we use `get_` all over the place in our codebase... 15 in this file alone and 355 across the whole of the Rust code. I'm torn between updating these to follow the convention and leaving them as-is to match our existing patterns.

Reply:
Huh. I err on the side of following the pattern especially if it's in the same file. But ideally we'd just update the existing violations too :)

Reply:
Let's leave that for another change; while I could update everything in this file it would trickle into callers and then the rest of the code would be half converted.